operator/validator/controller.go

```diff
@@ -915,11 +933,8 @@ func (c *controller) startValidator(v *validator.Validator) (bool, error) {
 		validatorErrorsCounter.Add(c.ctx, 1)
 		return false, errors.Wrap(err, "could not start validator")
 	}
-	if started {
-		c.recentlyStartedValidators++
-	}
 
-	return true, nil // TODO: what should be returned if c.validatorStart(v) returns false?
+	return started, nil
```

Comment:
I guess now `return c.validatorStart(v)` should be enough. but we should move the metric error counter higher - `		validatorErrorsCounter.Add(c.ctx, 1)`